integration_test/app.json

```diff
@@ -1,4 +1,5 @@
 {
   "name": "IntegrationTest",
-  "displayName": "IntegrationTest"
+  "displayName": "IntegrationTest",
+  "licenseKey": "ENTER_LICENSE_KEY"
```

Comment:
I added this new key here, to enable setting the license key easily for tests.
This can be later used for CI as well.